src/store/models/designer.ts

```diff
@@ -245,6 +266,31 @@ const designerModel: DesignerModel = {
       } else if (fromNode.type === 'bitcoin' && toNode.type === 'bitcoin') {
         // connecting bitcoin to bitcoin isn't supported
         return showError(l('linkErrBitcoin'));
+      } else if (fromNode.type === 'taro' || toNode.type === 'taro') {
+        if (fromNode.type === 'taro' && toNode.type === 'taro') {
+          //connecting taro to taro isn't supported
+          return showError(l('linkErrTarod'));
+        }
+        const taroName = fromNode.type === 'taro' ? fromNodeId : toNodeId;
+        const lndeName = fromNode.type === 'taro' ? toNodeId : fromNodeId;
+        const lnNetworkNode = network.nodes.lightning.find(
+          n => n.name === lndeName && n.implementation === 'LND',
+        ) as LndNode;
+        if (!lnNetworkNode) {
+          return showError(l('linkErrLNDImplementation', { nodeName: lndeName }));
+        }
+        const taroData = getTarodFilePaths(taroName, network);
```

Comment:
You don't need to call `getTarodFilePaths` because the path is already defined on the node in `network.nodes.taro[]`. The `node.path.adminMacaroon` field.